src/Types/Json.php

```diff
@@ -36,7 +36,11 @@ public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $pla
      */
     public function convertToPHPValue($value, AbstractPlatform $platform)
     {
-        return $value === null ? null : parent::convertToPHPValue($value, $platform);
+        if ($value === null || $value === '') {
+            return null;
```

Comment:
I think this was actually a conscious decision and would break apps if we change it.